apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.ts

```diff
@@ -33,6 +37,10 @@ export class DotToolbarUserComponent implements OnInit {
         this.loginService.watchUser((auth: Auth) => {
             this.auth = auth;
         });
+
+        this.haveLoginAsPermission$ = this.loginService
+            .getCurrentUser()
+            .pipe(map((res: CurrentUser) => res.loginAs));
```

Comment:
take(1) ?

Reply:
No because this is async piped so Angular takes care.